### PR TITLE
Make flake8 happy

### PR DESCRIPTION
**Pull request by aaaandrzej**
# Pull Request Template

**script name** -

**What have you Changed**

I've fixed the code formatting in few places plus removed some obsolete code to make `flake8` happy as I could see that there's an action in CI to check that, and it often fails.

### Self Check(Tick After Making pull Request)

- [ ] This issue was assigned to me.
- [x] One Change in one Pull Request
- [x] My file is in proper folder
- [x] I am following clean code and Documentation and my code is well linted with flake8.
- [ ] I have added README.md and requirements.txt with my script

If issue was not assigned to you Please don't make a PR. It will marked as invalid.

Join Us on Discord:- https://discord.gg/JfbK3bS

### CODE REVIEW

File: audiobook/config.py [hunk 1]
```diff
@@ -1,5 +1 @@
-supported_file_types = (".pdf", ".txt", ".epub", ".docx", ".doc")
```
Comment (aaaandrzej) [inline audiobook/config.py:1]: Removed as not used anymore

File: audiobook/main.py [hunk 2]
```diff
@@ -155,5 +172,5 @@ def read_book(self, input_book_path, password=None):
             elif user_input.isnumeric():
                 start_page = int(user_input) - 1
             else:
-                user_input = input("Please Select an option: \n 1. Type 'r' to read again: \n 2. Type 'p' to read previous page\n 3. Type 'n' to read next page\n 4. Type 'q' to quit:\n 5. Type page number to read that page:\n")
+                user_input = input(input_message)
```
Comment (aaaandrzej) [inline audiobook/main.py:175]: Avoiding code repetition

File: audiobook/__init__.py [hunk 3]
```diff
@@ -1 +0,0 @@
-from audiobook.main import AudioBook
```
Comment (DrakeEntity) [inline audiobook/__init__.py:1]: It's required when we use the Audiobook PIP module. otherwise, it will not import.

Comment (aaaandrzej) [reply audiobook/__init__.py:1]: Sure thing, should be good for both PIP and flake8 now.

**Review by DrakeEntity — APPROVED** [review]
Thanks for making it better.

**Pull request merged**